src/CosmosDB/CosmosDB/Az.CosmosDB.psd1

```diff
@@ -0,0 +1,141 @@
+#
+# Module manifest for module 'Az.CosmosDB'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 5/29/2019
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '1.0.0'
+
+# Supported PSEditions
+CompatiblePSEditions = 'Core', 'Desktop'
+
+# ID used to uniquely identify this module
+GUID = '91832aaa-dc11-4583-8239-bce5fd531604'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - CosmosDB service cmdlets for Azure Resource Manager in Windows PowerShell and PowerShell Core.
+
+For more information on CosmosDB, please visit the following: https://docs.microsoft.com/azure/cosmos-db/'
+
+# Minimum version of the PowerShell engine required by this module
+PowerShellVersion = '5.1'
+
+# Name of the PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+DotNetFrameworkVersion = '4.7.2'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# CLRVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.6.0'; })
```

Comment:
rebasing the latest master is recommended

Reply:
Please check if this looks good now.

Reply:
> Please check if this looks good now.

Please refer to Az.Accounts of 1.6.6

Reply:
Manually changed the Module Version to 1.6.5, with that change, I get this error: 

```
> Import-Module : The required module 'Az.Accounts' is not loaded. Load the module or remove the module from 'RequiredModules' in the file
> 'D:\az2\artifacts\Debug\Az.CosmosDB\Az.CosmosDB.psd1'.
> At line:1 char:1
> + Import-Module .\artifacts\Debug\Az.CosmosDB\Az.CosmosDB.psd1
> + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
>     + CategoryInfo          : ResourceUnavailable: (D:\az2\artifact...z.CosmosDB.psd1:String) [Import-Module], MissingMemberException
>     + FullyQualifiedErrorId : Modules_InvalidManifest,Microsoft.PowerShell.Commands.ImportModuleCommand
```

Reply:
> Manually changed the Module Version to 1.6.5, with that change, I get this error:
> 
> ```
> > Import-Module : The required module 'Az.Accounts' is not loaded. Load the module or remove the module from 'RequiredModules' in the file
> > 'D:\az2\artifacts\Debug\Az.CosmosDB\Az.CosmosDB.psd1'.
> > At line:1 char:1
> > + Import-Module .\artifacts\Debug\Az.CosmosDB\Az.CosmosDB.psd1
> > + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> >     + CategoryInfo          : ResourceUnavailable: (D:\az2\artifact...z.CosmosDB.psd1:String) [Import-Module], MissingMemberException
> >     + FullyQualifiedErrorId : Modules_InvalidManifest,Microsoft.PowerShell.Commands.ImportModuleCommand
> ```

How do you import the acccount module? Where do you get the error?